VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+
+
+## Technical requirements 
+
+### What are hardware requirements?
+
+Validators should expect to provision one or more data center locations with redundant power, networking, firewalls, HSMs and servers.
+We expect a modest level of hardware specifications will be needed initially and requirements to rise as network use rises. Participating in the testnet is the best way to learn more.
+
+### What are software requirements?
+
+In addition to running a cosmos node, validators should develop monitoring, alerting and management solutions.
+
+### What are bandwidth requirements?
+
+The Cosmos network has the capacity for very high throughput relative to chains like Ethereum or Bitcoin.
+We recommend that the data center nodes only connect to trusted full-nodes in the cloud or other validators that know each other socially. This relieves the data center node from the burden of mitigating Denial of Service attacks and syncing new nodes on the network.
+Ultimately, as the network becomes more heavily used, multigigabyte per day bandwidth is very realistic.
+
+### What does running a validator imply in terms of logistics? 
+
+A successful validator operation will require the efforts of multiple highly skilled individuals and continuous operational attention. This will be considerably more involved than running a bitcoin miner for instance.
+
+### How to handle key management?
+
+Validators should expect to run an HSM that supports ed25519 keys. Here are the current options:
+* YubiHSM 2
+* Ledger Nano S
+* Potential options
+* Ledger BOLOS SGX enclave
+* Thales nCypher support
+* Tendermint SGX enclave
+
+### What can validators expect in terms of operations? 
+
+Running effective operation is the key to avoiding unexpectedly unbonding or being slashed. This includes being able to respond to attacks, outages etc. Maintain security and isolation in your data center etc.
+
+### What are the maintenance requirements?
+
+Validators should expect to perform regular software updates to accommodate upgrades and bug fixes. There will inevitably be issues with the network early in its bootstrapping phase that will require substantial vigilance 
+
+### How can validators protect themselves from denial-of-service attacks?
+ 
+DDOS attacks occur when an attacker sends a flood of internet traffic to an IP address to prevent the server at the IP address from connecting to the internet.
+A DDOS attacker scan the network and learn the IP address of various validator nodes and try to disconnect them from communication with floods of traffic. 
```

Comment:
`scans and learns`

`and can try to`

`communication by flooding them with traffic.`

Reply:
ok